client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -18,31 +18,70 @@ import { updateTeam } from '../../actions/team';
 const ProfileInfoEdit = () => {
   const dispatch = useDispatch();
 
-  const { teamId, teamName, orgName, email } = useSelector(
+  const { teamId, teamName, orgName, email,password, confirmedPassword } = useSelector(
     (state) => state.team
   );
 
-  const [profileData, setInputs] = useState({ teamName, orgName, email });
+  const [profileData, setInputs] = useState({ teamName, orgName, email,password ,confirmedPassword});
 
   useEffect(() => {
-    setInputs({ teamName, orgName, email });
-  }, [email, orgName, teamName]);
+    setInputs({ teamName, orgName, email,password,confirmedPassword  });
+  }, [email, orgName, teamName,password,confirmedPassword]);
 
   const updateInputs = (form) => {
     const { name, value } = form.target;
-    setInputs({ ...profileData, [name]: value });
+    if (name !== "email" ){
+      setInputs({ ...profileData, [name]: value });
+    }
   };
+  const checkPassword = function(){
 
-  const [validated, setValidated] = useState(false);
-  const handleUpdate = (event) => {
-    const form = event.currentTarget;
-    event.preventDefault();
-    if (form.checkValidity() === false) {
-      event.stopPropagation();
-    } else {
-      dispatch(updateTeam(teamId, profileData));
+    if (! ({...profileData}.password === {...profileData}.confirmedPassword)){
```

Comment:
I think you can just simply use 
`if (profileData.password !== profileData.confirmedPassword)`

Reply:
well, its the same thing though so should be fine as is

Reply:
please address this and rest of the comments.

Reply:
No, it is not fine. It is technical debt. I agree with @pmyip3 and you should change it :)